uint8 left_rumble
uint8 right_rumble
---
int8 rc
